org-formation/050-costs/program-code-rules.json

```diff
@@ -0,0 +1,399 @@
+[
+  {
+    "Type": "REGULAR",
+    "Value": "Platform Infrastructure",
+    "Rule": {
+      "And": [
+    {
+      "Dimensions": {
+        "Key": "LINKED_ACCOUNT",
+        "Values": [
+          "531805629419",
+          "140124849929",
+          "254940560790",
+          "231505186444",
+          "153370007719",
+          "087547153226",
+          "797640923903",
+          "867686887310",
+          "420786776710",
+          "745159704268",
+          "325565585839",
+          "383874245509",
+          "449435941126",
+          "649232250620",
+          "804034162148",
+          "743644221192"
+        ],
+        "MatchOptions": [
+          "EQUALS"
+        ]
+      }
+    },
+    {
+      "Tags": {
+        "Key": "CostCenter",
+        "MatchOptions": [
+          "ABSENT"
+        ]
+      }
+    },
+    {
+      "Tags": {
+        "Key": "CostCenterOther",
+        "MatchOptions": [
+          "ABSENT"
+        ]
+      }
+    }
+      ]
+    }
+  },
+  {
+    "Type": "REGULAR",
+    "Value": "Platform Infrastructure",
+    "Rule": {
+      "Tags": {
+    "Key": "CostCenter",
+    "Values": [
+      "Platform Infrastructure"
+    ],
+    "MatchOptions": [
+      "STARTS_WITH"
+    ]
+      }
+    }
+  },
+  {
+    "Type": "REGULAR",
+    "Value": "BMGF Ki",
+    "Rule": {
+      "Tags": {
+    "Key": "CostCenter",
+    "Values": [
+      "BMGF-Ki"
+    ],
+    "MatchOptions": [
+      "STARTS_WITH"
```

Comment:
Why "STARTS_WITH" instead of an exact match?

Reply:
In order to match tags with either `/` or `_`, and in some later cases to group subprograms under a single program category (e.g. NTAP NF, or Mobile Toolbox). But now that I'm thinking about it, it might be better to match on the numeric code.

Reply:
@jesusaurus is there an `ENDS_WITH` operator?  I'd rather match `CD2H-CU / 11330` on the unambiguous numeric code " 113300" (perhaps with a leading space) than on the human readable prefix, `CD2H-CU`, which may not be unique.